lib/routes/showstart/search.ts

```diff
@@ -25,48 +56,42 @@ async function handler(ctx: Context): Promise<Data> {
     const type = ctx.req.param('type') || '';
     const keyword = ctx.req.param('keyword') || '';
 
+    let items;
+    let title;
     switch (type) {
         case 'event':
-            return {
-                title: `${TITLE} - 搜演出 - ${keyword || '全部'}`,
-                link: HOST,
-                item: await fetchActivityList({ keyword }),
-            };
+            title = `${TITLE} - 搜演出 - ${keyword || '全部'}`;
+            items = await fetchActivityList({ keyword });
+            break;
```

Comment:
The current early return is better

Reply:
> The current early return is better

Okay,
My mistake, just config `allowEmpty: true` and parameter `type` is required.

Reply:
I think it is not the responsibility of RSS to search for and ensure that the id target page exists.
I'm not sure if it makes sense to track a keyword on the platform as an RSS feed. 
So, close or mr several options okay for me, up to maintainer.